db/migrate/20211229065913_create_items.rb

```diff
@@ -0,0 +1,10 @@
+class CreateItems < ActiveRecord::Migration[6.1]
+  def change
+    create_table :items do |t|
+      t.string  :name
+      t.integer :cost
```

Comment:
maybe it mekas more sense to have price in menus_items table (for example, price will be different in different seasons or maybe it will sale or something)